### PR TITLE
Add back SB A11y impls for Android on SB 14/15

### PR DESCRIPTION
**Pull request by hlwarriner**
Add back SB A11y impls for Android on SB 14/15
    
 b/370027229

### CODE REVIEW

**Comment by hlwarriner** [comment]
I did local builds on SB 15 and SB 16 but don't have a device to do any testing. I can sanity check that the AOSP Evergreen APK runs for SB 15 and SB 16 on an Android device early next week.

File: cobalt/media/base/drm_system.h [hunk 1]
```diff
@@ -254,7 +254,7 @@ class DrmSystem : public base::RefCounted<DrmSystem> {
   TicketToSessionUpdateMap ticket_to_session_update_map_;
 
   MediaMetricsProvider media_metrics_provider_;
-  
+
```
Comment (kaidokert) [inline cobalt/media/base/drm_system.h:257]: small nit, looks like this file doesn't need a change at all.

**Review by kaidokert — APPROVED** [review]
minus the small comment, lgtm. Thanks for fixing this !

File: cobalt/media/base/drm_system.h (hunk 1 above)
Comment (hlwarriner) [reply cobalt/media/base/drm_system.h:257]: I know the whitespace fix is unrelated but do you mind if I just keep that commit in the PR? The trailing whitespace hook was complaining about it on push, for some reason.

**Comment by hlwarriner** [comment]
> I did local builds on SB 15 and SB 16 but don't have a device to do any testing. I can sanity check that the AOSP Evergreen APK runs for SB 15 and SB 16 on an Android device early next week.

I just ran a SB 15 APK with this fix on a Sabrina and while the a11y API issues are resolved, there's another Starboard conflict for `SbLogIsTty`. We can upload a separate PR for that.

**Pull request merged**